consensus/polybft/consensus_runtime.go

```diff
@@ -1085,13 +1085,19 @@ func (c *consensusRuntime) BuildProposal(view *proto.View) []byte {
 func (c *consensusRuntime) InsertBlock(proposal []byte, committedSeals []*messages.CommittedSeal) {
 	fsm := c.fsm
 
-	block, err := fsm.Insert(proposal, committedSeals)
+	alreadyInserted, block, err := fsm.Insert(proposal, committedSeals)
 	if err != nil {
 		c.logger.Error("cannot insert proposal", "error", err)
 
 		return
 	}
 
+	if alreadyInserted {
+		c.logger.Info("cannot insert proposal it was alredy previously inserted")
```

Comment:
I would completely remove this log, as we already have similar one in the [blockchain.WriteBlock](https://github.com/0xPolygon/polygon-edge/blob/develop/blockchain/blockchain.go#L884)

```suggestion
```

Reply:
It is completely removed.